news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md

```diff
@@ -0,0 +1,194 @@
+
+
+### Front office
+* [#18103](https://github.com/PrestaShop/PrestaShop/pull/18103): Duplicate address when submitting a form with errors, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+### Tests
+* [#18309](https://github.com/PrestaShop/PrestaShop/pull/18309): Change test fixtures that need to be in the future, by [@jolelievre](https://github.com/jolelievre)
+
+
+## Code changes in modules, themes & tools
+
+
+### Changes in developer documentation
+* [#495](https://github.com/PrestaShop/docs/pull/495): Change node version as we now support 12.x LTS, by [@NeOMakinG](https://github.com/NeOMakinG)
+* [#494](https://github.com/PrestaShop/docs/pull/494): Fix typo mistake. Thank you [@gadnis](https://github.com/gadnis)
+
+
+### The PrestaShop open source project
+* [#30](https://github.com/PrestaShop/open-source/pull/30): Update channels.md, by [@ttoine](https://github.com/ttoine)
+
+
+### Faceted search module
+* [#166](https://github.com/PrestaShop/ps_facetedsearch/pull/166): remove trailing space, by [@matthieu-rolland](https://github.com/matthieu-rolland)
+* [#165](https://github.com/PrestaShop/ps_facetedsearch/pull/165): Release 3.5.0, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#164](https://github.com/PrestaShop/ps_facetedsearch/pull/164): Bump to 3.5.0, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#163](https://github.com/PrestaShop/ps_facetedsearch/pull/163): Update npm packages, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#154](https://github.com/PrestaShop/ps_facetedsearch/pull/154): Sort products: InStock, available with qty 0, OutOfStock. Thank you [@gadnis](https://github.com/gadnis)
+
+
+### Live translation
+* [#5](https://github.com/PrestaShop/ps_livetranslation/pull/5): Remove trailing space in wording, by [@matthieu-rolland](https://github.com/matthieu-rolland)
+
+
+### Core Weekly Generator tool
+* [#27](https://github.com/PrestaShop/core-weekly-generator/pull/27): Add ps_wirepayment repository to the list, by [@ttoine](https://github.com/ttoine)
+
+
+### Contact informations module
+* [#26](https://github.com/PrestaShop/ps_contactinfo/pull/26): Release v3.3.0, by [@jolelievre](https://github.com/jolelievre)
+
+
+### Email subscription module
+* [#51](https://github.com/PrestaShop/ps_emailsubscription/pull/51): Do not send voucher when customer signs up if he did not register for newsletter , by [@matthieu-rolland](https://github.com/matthieu-rolland)
+
+
+### Theme customization
+* [#23](https://github.com/PrestaShop/ps_themecusto/pull/23): Bump version 1.1.1, by [@jolelievre](https://github.com/jolelievre)
+* [#22](https://github.com/PrestaShop/ps_themecusto/pull/22): Update references of strings in translations, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+### Prestashop UI Kit
+* [#90](https://github.com/PrestaShop/prestashop-ui-kit/pull/90): Update webpack from 3.22 to 4.42, by [@NeOMakinG](https://github.com/NeOMakinG)
```

Comment:
FYI this PR allows us to support Node.js 11 and 12 (previously we only spported Node.js v8 to v10)